pkgs/by-name/bi/bibiman/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  rustPlatform,
+  fetchCrate,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "bibiman";
+  version = "0.11.0";
+
+  src = fetchCrate {
+    inherit pname version;
+    hash = "sha256-b684N4QFpj/bpDF3XdpUACPXQBXUFOiy3To7y1GY1ug=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-RWLKiYyUQUgAPPWTugp1G05X3B9SbknmpVLc7eSdn8U=";
+
```

Comment:
add nix-update-script, versionCheckHook

Reply:
I add those in my last commit, hope it's good for you :+1: